### PR TITLE
adds a function to calculate time of peak delF for one pulse

### PR DESCRIPTION
**Pull request by aphilbrook**
adds a function to calculate time of peak delF for one pulse

### CODE REVIEW

File: scripts/time_to_peak.R [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#time to peak delF response for 1 pulse
+#open PlotGCaMP_multi csv file
```
Comment (mikeod38) [inline scripts/time_to_peak.R:2]: I'd just indicate here that this is intended for multi-pulse files. You could potentially make it more flexible and calculate for 2nd, 3rd etc.. but that's optional.

File: scripts/time_to_peak.R [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+time_to_peak <- function(genotype, #mandatory genotype, should be text
+                                 pulse_length = 60, # time of pulse in seconds
```
Comment (mikeod38) [inline scripts/time_to_peak.R]: 'pulse' sounds a little too similar to stimulus imo, maybe change to pulse-loop or loop-length?

File: scripts/time_to_peak.R (hunk 1 above)
Comment (aphilbrook) [reply scripts/time_to_peak.R:2]: done

File: scripts/time_to_peak.R (hunk 2 above)
Comment (aphilbrook) [reply scripts/time_to_peak.R]: done

**Pull request merged**